Installing the same file twice causes an error in Automake 1.11

JL 19/03/10

Upstream-Status: Pending

Index: git/clutter/Makefile.am
===================================================================
--- git.orig/clutter/Makefile.am	2010-03-19 11:37:51.000000000 +0000
+++ git/clutter/Makefile.am	2010-03-19 11:50:46.744042007 +0000
@@ -219,7 +219,6 @@
 	$(source_h) \
 	clutter-json.h \
 	clutter-enum-types.h \
-	clutter-version.h \
 	clutter.h
 
 DISTCLEANFILES = \
